GG/GG/TextControl.h

```diff
@@ -173,6 +173,12 @@ class GG_API TextControl : public Control
     virtual void SetText(const std::string& str,
                          const std::vector<boost::shared_ptr<Font::TextElement> >&text_elements);
 
+    /** Change TextControl's text to replace the text at templated \p targ_offset with \p new_text.
+
+        This does not reparse the TextControl's text. It is faster than SetText on a new
+        string. It will not find white space in the inserted text. */
+    void ChangeTemplatedText(const std::string& new_text, size_t targ_offset);
```

Comment:
Description and name are unclear. Does this insert text and push the following back to make room? Or replace the text of one text element starting immediately before/after the offset? Or replace all text start from the specified offset?